internal/manifests/manifests.go

```diff
@@ -91,6 +91,11 @@ func (m *Manifests) CreateClusterManifestInternal(ctx context.Context, params op
 		if !json.Valid(manifestContent) {
 			return nil, common.NewApiError(http.StatusBadRequest, errors.New("Manifest content has an illegal JSON format"))
 		}
+	} else if strings.HasSuffix(fileName, ".yaml.patch") || strings.HasSuffix(fileName, ".yml.patch") {
```

Comment:
IIUC, users have no way to give us 2 or more patch files that patch the same file, is that right? Can we come up with a design that will allow this? Maybe allow arbitrary descriptions after `_patch`? Something like `<original_name>.yaml.patch_fix_schedulable_masters`

Reply:
These can then be sorted alphabetically to enforce ordering, which would be a bonus

Reply:
I like this, thanks for the suggestion, thanks. Let me implement this on this PR.